cpp/modmesh/buffer/SimpleArray.hpp

```diff
@@ -261,7 +261,22 @@ class SimpleArrayMixinCalculators
             acopy[i] = athis->at(sidx);
             ++i;
         } while (athis->next_sidx(sidx));
-        return median_op(acopy);
+        return simd::generic::median(acopy.begin(), acopy.end());
+    }
+
+    value_type parallel_median() const
```

Comment:
It seems to me that you are not following the naming convention used in this file.  I am not sure whether or not you have read through the code.  If you have not, please check the related code first before adding inconsistent code.

If you have read through the code and decided to use a different style, please provide your argument and discuss.  It is not apparent to me why a different naming style is needed for your addition.

In my experience, it is not right to name a function using SIMD as "parallel" something, which usually implies to use multi cores or machines.